### ISSUE TITLE
non-Cartesian using batch size > 2

### ISSUE THREAD

**Issue by yongwanlim** [utcsilab/deepinpy#13]
Hi Jon, 

In deepinpy/deepinpy/forwards/mcmri/mcmri.py after line 82, I got an issue when using batch size > 2 for a non-Cartesian case.   

I am not sure if this is only my problem but I found a workaround as follow:

```
else: 
     out_list = []
     out_list.append(out0)
     for i in range(1, batch_size):
          out_list.append(self.Aop_adjoint_list[i](x[i]))
     out = torch.stack(out_list, dim=0)'
     return out
```

And if I use batch size >> 1, the gradient seems to be exploding... I found the loss function defined as self._loss_fun = torch.nn.MSELoss(reduction='sum') in deepinpy/deepinpy/recons/recon.py. I was wondering if the loss function should be normalized by the batch size so that the actual learning rate is kept consistent with different batch sizes?

Thanks!

**Comment by jtamir** [comment]
Hi, thanks. I think this is related to #5 and I forgot to change it. Regarding the exploding loss, maybe you need a smaller step size? Nonetheless, it would be great to include a more flexible mechanism for changing the loss, either through a config or through some other Python file that gets loaded. For now, I can add a config parameter.

Are you able to make a pull request for the batch_size issue?
